### ISSUE TITLE
chore(semver-changelog): automate CHANGELOG and semver bumps

### ISSUE THREAD

**Issue by gregswindle** [gregswindle/semver-demo#6]
## User story
As a contributor,
I want to automate CHANGELOG updates and semver bumps according to the types of changes I make
In order to expedite product delivery with consistency.

## Acceptance criteria
- [x] 1. [`commitplease`](https://github.com/jzaefferer/commitplease) validates commit messages with a pre-commit hook.
- [x] 2. [`standard-version`](https://github.com/conventional-changelog/standard-version) automates versioning and CHANGELOG generation using GitHub's squash button and [`conventional commit messages`](https://conventionalcommits.org/).
- [x] 3. Add release instructions to `README.md`.

## Notes

* Implement the [`commitplease` AngularJS](https://github.com/jzaefferer/commitplease#angularjs) configuration guidelines.
* Implement the [`standard-version` "CLI Usage" instructions](https://github.com/conventional-changelog/standard-version#cli-usage).

**Comment by gregswindle** [comment]
:trophy: Functional test for `standard-version` passes:

```sh

$ npm run release -- --dry-run

> semver-demo@0.0.0 release /Users/swindle/Projects/github/gregswindle/semver-demo
> standard-version "--dry-run"

✔ bumping version in package.json from 0.0.0 to 0.1.0
✔ created CHANGELOG.md
✔ outputting changes to CHANGELOG.md

---
<a name="0.1.0"></a>
# 0.1.0 (2017-06-21)


### Features

* **qa-gates:** automate travis-ci qa-gates and security checks ([#5](https://github.com/gregswindle/semver-demo/issues/5)) ([8d82c46](https://github.com/gregswindle/semver-demo/commit/8d82c46)), closes [#2](https://github.com/gregswindle/semver-demo/issues/2)
---

✔ committing package.json and CHANGELOG.md
✔ tagging release 0.1.0
ℹ  Run `git push --follow-tags origin master; npm publish` to publish


```

**Comment by gregswindle** [comment]
PR #7 merged successfully.
